bitwarden/__init__.py

```diff
@@ -0,0 +1,139 @@
+# -*- coding: utf-8 -*-
+
+"""'rbw' wrapper extension."""
+
+#  Copyright (c) 2022 Manuel Schneider
+
+import fnmatch
+import os
+from subprocess import run
+
+from albert import *
+
+md_iid = "0.5"
+md_version = "0.5"
+md_name = "Bitwarden"
+md_description = "rbw wrapper extensions"
+md_trigger = "bw "
+md_license = ""
+md_url = ""
+md_maintainers = "Asger Hautop Drewsen"
+md_bin_dependencies = ["rbw", "xclip"]
```

Comment:
we have [this ](https://github.com/albertlauncher/plugins/blob/master/python/README.md#setclipboardtexttextstr) please use it instead of xclip